src/util/loanSearch/dataUtils.js

```diff
@@ -61,13 +62,14 @@ export async function runLoansQuery(apollo, loanSearchState, origin) {
 export async function fetchLoanFacets(apollo) {
 	try {
 		const result = await apollo.query({ query: loanFacetsQuery, fetchPolicy: 'network-only' });
+		const resultEnums = await apollo.query({ query: loanEnumsQuery, fetchPolicy: 'network-only' });
```

Comment:
These can be sent off at the same time with `Promise.all`

Reply:
Fixed, thanks!